aioshelly/wsrpc.py

```diff
@@ -25,26 +29,68 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+def hex_hash(message: str) -> str:
+    """Get hex representation of sha256 hash of string."""
+    return hashlib.sha256(message.encode("utf-8")).hexdigest()
+
+
 @dataclass
-class RouteData:
-    """RouteData (src/dst) class."""
+class AuthData:
+    """RPC Auth data class."""
+
+    realm: str
+    username: str
+    password: str
+
+    def __post_init__(self) -> None:
+        """Call after initialization."""
+        self.ha1 = hex_hash(f"{self.username}:{self.realm}:{self.password}")
+        self.ha2 = hex_hash("dummy_method:dummy_uri")
```

Comment:
why generate this on `__post_init__` and not just statically set it? 

Let's add a comment why this is necessary.